h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/util/BoosterHelper.java

```diff
@@ -11,7 +13,7 @@
 public class BoosterHelper {
 
   public static Booster loadModel(InputStream in) throws XGBoostError, IOException {
-    return Booster.loadModel(in);
```

Comment:
was using a package-private method for no reason since XGBoost.loadModel exposes it